include/glow/Graph/Graph.h

```diff
@@ -191,12 +191,12 @@ class Function final : public Named {
 
   ConvolutionNode *createConv(llvm::StringRef name, NodeValue input,
                               size_t depth, size_t kernel, size_t stride,
-                              size_t pad, size_t group);
+                              llvm::ArrayRef<size_t> pads, size_t group);
```

Comment:
Maybe it makes sense to keep another `createConv()` that takes a `size_t pad` and just internally creates equal padding for top, left, bottom, right? Instead of requiring changing every `createConv()`, since it seems that equal padding is more common?